.buildkite/macos/config.yml

```diff
@@ -0,0 +1,20 @@
+name: ray-postmerge-macos
```

Comment:
this makes me thinking.. we can actually also put the postmerge config file here in the repo too, and then we just need one builtin config in rayci

not high priority and optional though.

Reply:
i think we should yes, let's do in another PR